release/smbchk.py

```diff
@@ -45,6 +45,13 @@
     'cyclus::Composition::NewDecay',
 }
 
+cpp11_symbols = [
```

Comment:
Also maybe this variable name could have the word "replacements" in it, since that is what it is.